File: Modul-6/Soal-3/PRAK603-2410817310012-FAISALTANJUNG.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
        return 0;
    }
    int arr1[n1], arr2[n2];
    for (int i = 0; i < n1; i++) {
        scanf("%d", &arr1[i]);
    }
    for (int i = 0; i < n2; i++) {
        scanf("%d", &arr2[i]);
    }

    for (int i = 0; i < n1; i++) {
        printf("%d ", arr1[i] * arr2[i]);
    }
    return 0;
}

File: Modul-6/Soal-3/PRAK603-2410817310012-FAISALTANJUNG.py
n1, n2 = map(int, input().split())

if n1 != n2:
    print("Jumlah tidak sama")
else:
    arr1 = list(map(int, input().split()))
    arr2 = list(map(int, input().split()))
    result = [arr1[i] * arr2[i] for i in range(n1)]
    print(" ".join(map(str, result)))
